modules/ROOT/pages/_partials/data-sync/server-offline-and-conflict.adoc

```diff
@@ -108,164 +64,104 @@ type Mutation {
-
-* `conflictHandler.resolveOnServer` is called with the `clientWins` strategy. In this case, the `resolvedState` will be the new data provided by the client. The newly `resolvedState` should be persisted.
-* `conflictHandler.resolveOnServer` also returns a `response`, which should be returned to the client.
-
-The `response` object is a `ConflictResolution` object that tells the client there was a conflict, that it was resolved on the server and provides the new `resolvedState`. In most cases, the client needs to know about conflicts that happen on the server. This allows the client to handle the conflict accordingly. For example, the screen the user is looking at might need to be refreshed with new data after a conflict.
-
-== Conflict Resolution Strategies
-
-There is one default conflict resolution strategy.
+. When using the `HashObjectState` implementation you must provide your own hashing function. The function signature should be as follows:
 
-* `clientWins` - This strategy accepts the data provided by the client.
-
-== Conflict Resolution using the `reject` Function
-
-It is possible to implement a 'server wins' style strategy using the `reject` method. This is useful in conflict cases where we want to reject the client's changes and force the client to use the latest data stored on the sever.
-
-[source,javascript]
++
+[source, javascript]
 ----
-// If a conflict is detected, call the reject function
-// to keep the server side data and force the client to update
-if (conflictHandler.hasConflict(serverData, clientData)) {
-  return conflictHandler.reject(task, clientData)
+const hashFunction = (object) {
+  // Using the Hash library of your choice
+  const hashedObject = Hash(object)
+  // return the hashedObject in string form
+  return hashedObject;
 }
-// otherwise continue and perform the standard mutation logic
 ----
 
-== Custom Conflict Resolution Strategies
-
-In most real world cases, the conflict resolution strategies used by your application are custom and specific to your application's needs. Your application may deal with different conflicts in different ways. It is possible to implement a custom `ConflictResolutionStrategy` function to be used with `resolveOnServer`.
+. Provide this function when instantiating the `HashObjectState`:
 
-[source, javascript]
++
+[source,javascript]
 ----
-function customResolutionStrategy (serverState, clientState) {
-  return {
-    title: `${serverState.title} ${clientState.title}`
-  }
-}
+const conflictHandler = new HashObjectState(hashFunction)
 ----
 
-This example takes string values from the server and the client records, merges them together and returns the newly resolved object. This example is a little contrived but it shows how any strategy could be implemented.
+. Implement the resolver. Every conflict can be handled using a set of predefined steps, please see below for an example.
```

Comment:
. Implement the resolver. Every conflict can be handled using a set of predefined steps. Please see below for an example.